### PR TITLE
Feature/remove telescope events

### PR DESCRIPTION
**Pull request by jnation3406**
I don't think this pull request is very understandable, but mainly the changes were just made in the telescope_states.py file to use the mysql_telemetry index "Available For Scheduling Reason" datum for everything, rather than use the telescope_events index. The logic to group states is simplified because the datum wraps all states together and only changes when a state changes. The output is munged into the previous telescope_states format so that all upstream stuff that uses the endpoints (including our plots) doesn't need to have it's code modified.

### CODE REVIEW

File: valhalla/common/telescope_states.py [hunk 1]
```diff
@@ -52,15 +63,20 @@ def _get_available_telescopes(self):
...
                     "filter": [
+                        {
+                            "match": {
+                                "datumname": "Available For Scheduling Reason"
+                            }
+                        },
                         {
                             "range": {
                                 "timestamp": {
                                     # Retrieve documents 1 hour back to capture the telescope state at the start.
```
Comment (Fingel) [inline valhalla/common/telescope_states.py]: Might want to update this comment.

File: valhalla/common/telescope_states.py [hunk 2]
```diff
@@ -103,108 +119,83 @@ def _get_es_data(self, sites, telescopes):
...
-    def _set_lump(self, event):
+    def _create_lump(telcode, type, reason, start):
         return {
-            'reasons': [event['_source']['reason']],
-            'types': [event['_source']['type']],
-            'start': string_to_datetime(event['_source']['timestamp']),
-            'telescope': self._telescope(event['_source']),
-            'latest_timestamp': string_to_datetime(event['_source']['timestamp'])
+            'telescope': telcode,
+            'event_type': type,
```
Comment (Fingel) [inline valhalla/common/telescope_states.py]: Try to avoid using builtins whenever possible.

Comment (jnation3406) [reply valhalla/common/telescope_states.py]: good idea

File: valhalla/common/telescope_states.py (hunk 1 above)
Comment (jnation3406) [reply valhalla/common/telescope_states.py]: ok

File: valhalla/common/telescope_states.py [hunk 3]
```diff
@@ -4,9 +4,11 @@
 from datetime import datetime, timedelta
```
[the remark this answers is not in the document]
Comment (jnation3406) [reply valhalla/common/telescope_states.py]: ok

**Comment by jnation3406** [comment]
I realized this will break the telescope_states plot/api queries for times before the "Available For Scheduling Reason" Datum existed, so we are going to wait at least until the semester boundary, and maybe a little longer before merging this in.

**Pull request merged**